doc/classes/TileSet.xml

```diff
@@ -50,6 +104,53 @@
 			<argument index="0" name="id" type="int">
 			</argument>
 			<description>
+				Returns the size of the subtiles in an autotile.
+			</description>
+		</method>
+		<method name="autotile_get_spacing" qualifiers="const">
+			<return type="int">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<description>
+				Returns the spacing between subtiles of the autotile.
+			</description>
+		</method>
+		<method name="autotile_get_subtile_priority">
+			<return type="int">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<argument index="1" name="coord" type="Vector2">
+			</argument>
+			<description>
+				Returns the priority of the subtile from an autotile given its coordinates.
+				When more than one subtile has the same bitmask value, one of them will be picked randomly for drawing. Its priority will define how often it will be picked.
+			</description>
+		</method>
+		<method name="autotile_get_z_index">
+			<return type="int">
+			</return>
+			<argument index="0" name="id" type="int">
+			</argument>
+			<argument index="1" name="coord" type="Vector2">
+			</argument>
+			<description>
+				Returns the drawing index of the subtile from an autotile given its coordinates.
```

Comment:
Some of these methods also works for ATLAS_TILE, such as z-index, I think it's worth to mentioning to let the users know that, because the name of the method doesn't make that very clear.

Reply:
Good idea.